### PR TITLE
Error handling refactoring

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: src/components/common/RequestStatusBox.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Grid, Responsive } from 'semantic-ui-react'
+import styled from 'styled-components'
+
+const RequestStatusBoxGrid = styled(Grid)`
```
Comment (janedegtiareva) [inline src/components/common/RequestStatusBox.js]: Do we want to start adding docs for reusable components?

Comment (vgrichina) [reply src/components/common/RequestStatusBox.js]: What would you put in docs for this component?

Comment (janedegtiareva) [reply src/components/common/RequestStatusBox.js]: What it does and maybe a usage example (if we are relatively sure it's not going to change radically).

Comment (vgrichina) [reply src/components/common/RequestStatusBox.js]: It is hard to keep non-running usage examples up to date TBH. Tests prob work better for that purpose.

> if we are relatively sure it's not going to change radically

not sure it's going to survive introduction of design system

**Pull request merged**